packages/vite/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { basename, extname } from 'node:path'
+import { basename, extname, join } from 'node:path'
```

Comment:
you can't use the `join` from `path` because it will output `\` on Windows. You need the one from `path.posix` so that you always get `/`

Reply:
But `import { join } from 'node:path/posix'` should be ok then?

Reply:
yep! looks good now. thanks